api/src/integration/java/org/pmiops/workbench/google/DirectoryServiceImplIntegrationTest.java

```diff
@@ -47,8 +47,14 @@ public void testDirectoryServiceUsernameIsTaken() {
   @Test
   public void testCreateAndDeleteTestUser() {
     String userName = String.format("integration.test.%d", Clock.systemUTC().millis());
-    service.createUser("Integration", "Test", userName, "notasecret");
+    service.createUser("Integration", "Test", userName, "notasecret@gmail.com");
     assertThat(service.isUsernameTaken(userName)).isTrue();
+    // Ensure our two custom schema fields are correctly set & re-fetched from GSuite.
+    assertThat(service.getContactEmailAddress(userName).equals("notasecret@gmail.com"));
+    assertThat(
+            service.getUserByUsername(userName).getCustomSchemas()
+                    .get("All_of_Us_Workbench")
+                    .get("Institution").equals("All of Us Research Workbench"));
```

Comment:
nit: spacing. I believe we're using 2-space tabs in these cases.

Reply:
Gotcha. Is there some way I can set up IntelliJ (& document / provide a standard config for the team), or run some formatting script which allows me to not have to think about this?

Reply:
Would love to get auto-formatting setup in this project - right now our rules are extremely loosely defined, and only automatically checked in Codacy (by UI config only) , which is a huge pain and not config-driven.

Possibly:
- https://github.com/google/google-java-format
- https://clang.llvm.org/docs/ClangFormat.html

Reply:
Fixed the spacing here. I'll look into auto-format sometime next week, thanks for the pointers!